doc/classes/ProjectSettings.xml

```diff
@@ -443,7 +443,7 @@
 		</member>
 		<member name="debug/settings/fps/force_fps" type="int" setter="" getter="" default="0">
 			Maximum number of frames per second allowed. The actual number of frames per second may still be below this value if the game is lagging.
-			If [member display/window/vsync/use_vsync] is enabled, it takes precedence and the forced FPS number cannot exceed the monitor's refresh rate.
+			If [member display/window/vsync/vsync_mode] is set to ENABLED or ADAPTIVE, it takes precedence and the forced FPS number cannot exceed the monitor's refresh rate.
```

Comment:
```suggestion
			If [member display/window/vsync/vsync_mode] is set to [code]ENABLED[/code] or [code]ADAPTIVE[\code], it takes precedence and the forced FPS number cannot exceed the monitor's refresh rate.
```

Reply:
Done, but changed to "[code]Enabled[/code] or [code]Adaptive[/code]" to match with the actual text in the dropdown menu.